src/serverExplorer.ts

```diff
@@ -950,7 +950,14 @@ export class ServerExplorer implements TreeDataProvider<RSPState | ServerStateNo
             return Array.from(this.RSPServersStatus.values()).map(rsp => rsp.state);
         } else if (this.isRSPElement(element) && (element as RSPState).serverStates !== undefined) {
             // rsp parent -> return servers
-            return (element as RSPState).serverStates;
+            const zzz = 33;
+            const abc = zzz+212;
+            const qqq = abc+3;
+            console.log(qqq);
```

Comment:
@robstryker, thanks for implementing this request of mine so quickly. But it looks like some extra debugging lines were left behind here?